docs/installing.rst

```diff
@@ -12,15 +12,21 @@ Prebuilt Packages
 
 Installing the prebuilt hipRAND packages requires a ROCm-enabled platform. See the `ROCm documentation <https://docs.amd.com/>`_ for more information. After installing ROCm or enabling the ROCm repositories, hipRAND can be obtained using the system package manager.
```

Comment:
```suggestion
Installing the prebuilt hipRAND packages requires a ROCm-enabled platform. See the `ROCm documentation <https://rocm.docs.amd.com/>`_ for more information. After installing ROCm or enabling the ROCm repositories, hipRAND can be obtained using the system package manager.
```

Update the URL to ROCm docs.  doc.amd.com redirect to rocm.docs.amd.com will be removed soon.

Reply:
Done as requested.